2-mandatory/3-financial-times.js

```diff
@@ -5,33 +5,73 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  // TODO
+  let newArrayTitle = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length <= 65) {
+      newArrayTitle.push(allArticleTitles[i]);
+    }
+  }
+  return newArrayTitle;
 }
+    
+
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
-}
+  let newString = ''
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (newString.length < 1 || newString.split(" ").length > allArticleTitles[i].split(" ").length) {
+      newString = allArticleTitles[i];
+    }
+
+  }
 
+return newString;
+}
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let newClicks = [];
+
+    for (let i = 0; i < allArticleTitles.length; i++) {
+        for (let letter of allArticleTitles[i])  {
+            if (letter === "0"||
+                letter === "1"||
+                letter === "2"||
+                letter === "3"||
+                letter === "4"||
+                letter === "5"||
+                letter === "6"||
+                letter === "7"||
+                letter === "8"||
+                letter === "9") {
+                newClicks.push(allArticleTitles[i])
+            }
```

Comment:
I like that you've solved the problem correctly with the ways you knew 🥇 

There is a function `isNaN` that can check if a string is not a number, which can help to do this step for you.